docs/root/intro/version_history.rst

```diff
@@ -83,6 +83,7 @@ Version history
 * tls: added support for CRLs in :ref:`trusted_ca <envoy_api_field_auth.CertificateValidationContext.trusted_ca>`.
 * tls: added support for :ref:`multiple server TLS certificates <arch_overview_ssl_cert_select>`.
 * tls: added support for :ref:`password encrypted private keys <envoy_api_field_auth.TlsCertificate.password>`.
+* tls: added the ability to build :ref:`BoringSSL FIPS <arch_overview_ssl_fips>` using ``--define boringssl=fips`` Bazel option.
```

Comment:
Can you add the new option to https://github.com/envoyproxy/envoy/tree/master/bazel#enabling-optional-features perhaps?